releases/v2.1.2.md

```diff
@@ -0,0 +1,121 @@
+---
+title: What&#39;s New in v2.1.2
+toc: true
+summary: Additions and changes in CockroachDB version v2.1.2 since version v2.1.1
+---
+
+## December 10, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v2.1.2
+~~~
+
+### Backward-incompatible changes
+
+- The `CHANGEFEED` [`experimental-avro` option](../v2.1/create-changefeed.html#options) has been renamed `experimental_avro`. [#32235][#32235]
+
+### SQL language changes
+
+- The [`IMPORT format (file)`](../v2.1/import.html) syntax is deprecated in favor of `IMPORT format file`. Similarly, `IMPORT TABLE ... FROM format (file)` is deprecated in favor of `IMPORT TABLE ... FROM format file`. [#31301][#31301]
+- bpchar OIDs are now allowed from client connections. [#32661][#32661]
```

Comment:
The type OID "bpchar" is now allowed to provide placeholder string values, for compatibility with PostgreSQL clients that use them.